.circleci/config.yml

```diff
@@ -115,7 +115,7 @@ references:
     name: Install npm dependencies
     command: |
       if [ ! -d node_modules ]; then
-        npm ci
+        npx lerna bootstrap
```

Comment:
Compared with `npm ci`, which will install packages according to lockfiles, lerna bootstrap should:
- run npm ci in all the packages by default in CI environment
- Symlink between any packages as need be

I don't think this caching scheme will be valid any longer 😞 

If the root packages.json and shrinkwrap are the same, we restore root node_modules and skip this command completely.

In a case where a packages was newly added and/or started to depend on another package in the repo, the cache would not be invalidated and the command would be skipped completely. That would not be the desired behavior.

We'll need to rethink how we cache this bit.